src/app.ts

```diff
@@ -0,0 +1,14 @@
+import { NestFactory } from '@nestjs/core';
+import { INestApplication } from '@nestjs/common';
+
+import { AppModule } from './app.module';
+
+export async function createApp(): Promise<INestApplication> {
+  const app = await NestFactory.create(AppModule, {
+    logger: process.env.APP_LOGGER as unknown as false,
+  });
+
+  app.setGlobalPrefix('/v1');
```

Comment:
Consider using a configuration file or environment variables for setting the global prefix '/v1'. This would make it more flexible if the prefix needs to be changed in the future. [medium]